dask/array/tests/test_sparse.py

```diff
@@ -34,8 +34,6 @@
     lambda x: da.transpose(x, (1, 2, 0)),
     lambda x: x.sum(),
     lambda x: x.mean(),
-    pytest.param(lambda x: x.std(),
```

Comment:
It seems you removed the entire test instead of the xfail.

Reply:
Oh damn, thanks for noticing that, let me add it back.